### ISSUE TITLE
No Joi options API for i18n language config

### ISSUE THREAD

**Issue by mytharcher** [AndrewKeig/express-validation#77]
As in `lib/index.js` at [line:69](https://github.com/AndrewKeig/express-validation/blob/master/lib/index.js#L69)

~~~js
  var joiOptions = {
    context: context || request,
    allowUnknown: allowUnknown,
    abortEarly: false
    // , language: {...}
  };
~~~

May it configurable for the `joiOptions` which user could add other options like `language` for Joi?

I didn't find any approach or workaround to make it happen now, or did I miss anything?

**Comment by biblicalph** [comment]
@mytharcher Seems the options are rather limited here. I'd be happy to submit a PR to fix the above issue and possibly rewrite large sections of the app. I can work on it over the weekend.
Lemme know what you think

**Comment by mytharcher** [comment]
@biblicalph Sounds good. But the latest update of this repo is from a year ago. Not sure if the author will take care of it.

**Comment by AndrewKeig** [comment]
Closing, released 3.0
